packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -99,11 +106,41 @@ bool _isFlingGesture(Velocity velocity) {
   return speedSquared > kMinFlingVelocity * kMinFlingVelocity;
 }
 
+
+/// Defines a line between two pointers on screen.
+///
+/// [_LineBetweenPointers] is an abstraction of a line between two pointers in
+/// contact with the screen. Used to track the rotation of a scale gesture.
+class _LineBetweenPointers{
+
+  /// Creates a [_LineBetweenPointers]. None of the [pointerStartLocation], [pointerStartId]
+  /// [pointerEndLocation] and [pointerEndId] must be null. [pointerStartId] and [pointerEndId]
+  /// should be different.
+  _LineBetweenPointers({
+    this.pointerStartLocation = Offset.zero,
+    this.pointerStartId = 0,
+    this.pointerEndLocation = Offset.zero,
+    this.pointerEndId = 1
+  }) : assert(pointerStartLocation != null && pointerEndLocation != null),
+       assert(pointerStartId != null && pointerEndId != null),
+       assert(pointerStartId != pointerEndId);
+
+  /// The location and the id of the pointer that marks the start of the line,
+  final Offset pointerStartLocation;
+  final int pointerStartId;
+
+  /// The location and the id of the pointer that marks the end of the line,
+  final Offset pointerEndLocation;
+  final int pointerEndId;
+
+}
+
+
 /// Recognizes a scale gesture.
 ///
 /// [ScaleGestureRecognizer] tracks the pointers in contact with the screen and
-/// calculates their focal point and indicated scale. When a focal pointer is
-/// established, the recognizer calls [onStart]. As the focal point and scale
+/// calculates their focal point, indicated scale and rotation. When a focal pointer is
```

Comment:
nit: comma after `scale`

Reply:
As written, it means that the focal point is the indicated scale and rotation. What we're trying to say here is that the calculation is of the focal point and the scale and the rotation.

(It's the difference between "I met your neighbours, Alice and Bob" vs "I met your neighbours, Alice, and Bob". In the first case you met two people, in the second you met at least three, maybe more. In the first case it's equivalent to "I met Alice and Bob, [who are] your neighbours", in the second it's equivalent to "I met Alice, Bob, and your neighbours".)